### ISSUE TITLE
Getter methods for boolean types

### ISSUE THREAD

**Issue by ramonaborg** [kobylynskyi/graphql-java-codegen#1209]
Getter methods for boolean types generated for both classes and interfaces,  mirrored from the GraphQL schema are prefixed with get, however, the common practice is to have these prefixed with 'is'. 

For instance, getter method for the boolean property active should be generated as isActive() and not getActive() as illustrated in the code snipped below.

Currently generated as:

```
public class Animal {
  String getName();
  String getType();
  Boolean getAdopted();
}
```

Should be generated like this:

```
public class Animal {
  String getName();
  String getType();
  Boolean isAdopted();
}
```

**Comment by esfomeado** [comment]
The `is` convention applies only for the `boolean` primitive and not to `Boolean`.

Since the generated code produces a `Boolean` it should use `get` instead of `is`.

**Comment by ramonaborg** [comment]
Yes, you are right Boolean should be generated with ‘get’ and the primitive type boolean should be with ‘is’. In the docs, there is listed that when a property is non-nullable it should be generated with primitive types.  In this case the non-nullable boolean property should be generated with ‘is’

If we take into consideration the below code:

```
Interface ItemDefinition {
  Title: String!
  Description: String!
  Available: Boolean!
 } 
```
The code generated with the plugin will be like the below example (since the field is non-nullable it will use the boolean primitive type):

```
public Interface ItemDefinition {
  String getTitle();
  String getDescription();
  boolean getAvailable();   
 } 
```
